charts/judge-api/templates/deployment.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "judge-api.fullname" . }}
+  labels:
+    {{- include "judge-api.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "judge-api.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "judge-api.selectorLabels" . | nindent 8 }}
+    spec:
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          ## TODO: parameterize the command and/or update judge-api to mount envvars
+          args: ["serve", "--sql-connection", "root:root@tcp(judge-mysql.default.svc.cluster.local:3306)/judgeapi?parseTime=true", "--listen=tcp://0.0.0.0:8080", "--kratos-admin-url=http://kratos-admin.default.svc.cluster.local"]
```

Comment:
I think this will need to be done now. Otherwise, the chart is going to have to be modified by any consumers. Also, another argument for templating through the parent chart: the sql connection here will be the same as the one used by the archivista chart.

Reply:
we can do that at a later time in a new PR I think, too